src/plugins/core/sheet.ts

```diff
@@ -651,6 +651,9 @@ export class SheetPlugin extends CorePlugin<SheetState> implements SheetState {
     const { orderedSheetIds, sheets } = this;
     const name = cmd.name && cmd.name.trim().toLowerCase();
 
+    if (name && name.length > 100) {
```

Comment:
we could maybe take the same character limit as Excel, which is 31 chars.

Also I would flag this as an improvement unless you found a really blocking behaviour for the users.

Reply:
Should this now be merged into the master branch?

I've completed all the suggested changes :)

Reply:
let's target master, yes :)